12runningSS3.tex

```diff
@@ -295,15 +295,15 @@
 
 
 
-Running SS3 from within R may be desirable for setting up simulations where many runs of SS3 models are required (e.g., \href{https://github.com/ss3sim/ss3sim}{\texttt{ss3sim}}) or if \href{https://r4ss.github.io/r4ss/index.html}{\texttt{r4ss}} is already used to read model output.
+Running SS3 from within R may be desirable for setting up simulations where many runs of SS3 models are required (e.g., \href{https://github.com/ss3sim/ss3sim}{\texttt{ss3sim}}) or if \texttt{r4ss} is already used to read model output.
```

Comment:
Why did you remove the link to r4ss?

Reply:
There is a link to r4ss in the 14r4ss.tex file (there is a link to this section of the manual on line 169). It seemed potentially redundant to have the link there as well. Plus there is a link to the github.io site for r4ss in line 285. I just put in an additional sentence about further info on r4ss can be found in the r4ss section. I am happy to put the link to r4ss back in there though if you think it should be there!

Reply:
The manual is often not read as a book and instead users search for a relevant section and get the information that they need and close it out. I prefer if something is linked that it is linked in every instance but I think consistency is a bigger key here rather than doing what I think is best 🤣 because users should know where to find things based on their prior knowledge of looking for other things.

Reply:
Yeah I get that for sure. Okay, I will keep it out for now, but I am happy to add links in if helpful/needed in the future!